waspc/data/Generator/templates/sdk/wasp/server/email/core/providers/dummy.ts

```diff
@@ -1,7 +1,9 @@
 import { DummyEmailProvider, EmailSender } from "../types";
 import { getDefaultFromField } from "../helpers.js";
 
-const yellowColor = "\x1b[33m%s\x1b[0m";
```

Comment:
This isn't just yellow though, is it? This look smore complex, like two things? Ah I get it now, it is a format string, not just yellow color.